rosdep/base.yaml

```
@@ -6657,6 +6657,7 @@ matio:
   debian: [libmatio-dev]
   fedora: [matio-devel]
   nixos: [matio]
+  rhel: [matio]
```

Two suggestions:
1. This isn't available for RHEL 9 yet, so let's just add it for RHEL 8 (which is probably where you want it anyway)
2. The other packages appear to get development headers, so we should get them here too.

```suggestion
  rhel:
    '*': null
    '8': [matio-devel]
```

---

Awesome, thank you for the swift review and suggestions